target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -45,6 +45,15 @@ engenius,ens620ext)
 	ucidef_set_led_netdev "lan1" "LAN1" "green:lan1" "eth0"
 	ucidef_set_led_netdev "lan2" "LAN2" "green:lan2" "eth1"
 	;;
+mikrotik,hap-ac3)
+	ucidef_set_led_netdev "wan" "WAN" "green:wan" "eth1"
+	ucidef_set_led_netdev "ethernet" "ETHERNET" "green:ethernet" "eth1"
```

Comment:
ethernet and wan are linked to the same?

Reply:
I prefer sticking to the vendor behavior. If people want additional triggers based on preference, they should add them themselves.

Reply:
I got no idea how MikroTik triggers this LED at all.

Reply:
So you just assigned the LEDs based on what you thought would be good?

Reply:
I don't get it? You are setting an ethernet LED in your code? And internet LED is WAN LED? Why not just use the proper names so there is less confusion?

Reply:
> "ethernet" in ROS, so I kept that name and triggered it of the WAN as well.

And that's totally arbitrary, so this should be removed.

> In ROS they have some kind of internet detection, probably some kind of ping test to their server based on which that LED is triggered of.

So OEM does not signal pure WAN link state, like it does for LAN ports?

Reply:
> > "ethernet" in ROS, so I kept that name and triggered it of the WAN as well.
> 
> And that's totally arbitrary, so this should be removed.

Ok, fine will drop it.
> 
> > In ROS they have some kind of internet detection, probably some kind of ping test to their server based on which that LED is triggered of.
> 
> So OEM does not signal pure WAN link-state, like it does for LAN ports?
They do via the port LED and then they also signal internet reachability via the ethernet LED